homework-solutions/js/hw-4/task-3.js

```diff
@@ -6,3 +6,31 @@
   преобразовываясь в number
 
 */
+
+let minAge = 15;
+
+let maxAge = 60;
+
+let age;
+
+age = '3d';
+
+if (typeof age !== 'number') {
+  age = +age;
+}
+
+if (age) {
+  if (typeof age !== 'number') {
+    console.log('Incorrect data type');
+  } else if (age < minAge) {
+    console.log("You don't have access cause your age is " + age + " It's less then " + minAge);
+  } else if (age >= minAge && age < maxAge) {
+    console.log('Welcome  !');
+  } else if (age > maxAge) {
+    console.log('Keep calm and look Culture channel');
+  } else {
+    console.log('Technical work');
+  }
+} else {
+  console.log('Not a number');
+}
```

Comment:
И неплохо будет разобраться в отличиях между `isNaN | Number.isNaN`
```suggestion
if (isNaN(age)) {
  console.log('Incorrect data type');
} else {
  age = Number(age); // +age
  if (age < minAge) {
    console.log(
      "You don't have access cause your age is " +
        age +
        " It's less then " +
        minAge,
    );
  } else if (age >= minAge && age < maxAge) {
    console.log('Welcome  !');
  } else if (age > maxAge) {
    console.log('Keep calm and look Culture channel');
  } else {
    console.log('Technical work');
  }
}

```